pos_session_pay_invoice/models/account_journal.py

```diff
@@ -0,0 +1,13 @@
+from odoo import api, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    @api.model
+    def name_search(self, name="", args=None, operator="ilike", limit=100):
```

Comment:
I think this shouldn't be done at this level, but modifying the domain of the field `journal_id` in the wizard.

Reply:
Seen that this is working even if no typing.